corehq/apps/cloudcare/templates/cloudcare/includes/touchforms-inline.html

```diff
@@ -1,23 +1,18 @@
 {% load hq_shared_tags %}
 
 {# touchforms includes #}
-<script src="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-1.8.16.min.js' %}"></script>
-<link rel="stylesheet" href="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-redmond-1.8.16.css' %}">
-
-<script src="{% static 'formplayer/script/vendor/shortcut.js' %}"></script>
-<script src="{% static 'formplayer/script/vendor/markdown-it-4.2.0.min.js' %}"></script>
-<script src="{% static 'formplayer/script/vendor/ba-tiny-pubsub.min.js' %}"></script>
-<script src="{% static 'formplayer/script/vendor/purify.js' %}"></script>
-<script src="{% static 'formplayer/script/xforgasm.js' %}"></script>
-<script src="{% static 'formplayer/script/formnav-all.js' %}"></script>
-<script src="{% static 'formplayer/script/entrycontrols_full.js' %}"></script>
-<script src="{% static 'formplayer/script/fullform-ui.js' %}"></script>
-
-<link rel="stylesheet" href="{% static 'formplayer/style/webforms.css' %}">
-
-
-
-<script src="{% static "cloudcare/js/util.js" %}"></script>
-<script src="{% static 'cloudcare/js/touchforms-inline.js' %}"></script>
+<link rel="stylesheet" href="{% new_static 'jquery-ui/themes/base/jquery-ui.min.css' %}">
```

Comment:
hmm....some css files among the JS. In one place this is included in `<head>`, but in another it's in the body/partial. Not a good practice to do this, as it forces the browser to re-generate ALL the styles on reload / partial reload. Also isn't jquery UI now a decorator too?